2024-09-10 18:15:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 18:15:43 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: ; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription(255)); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.mycompany.adventure.Adventure.newDB(Adventure.java:343)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:323)
	at com.mycompany.adventure.Adventure.init(Adventure.java:27)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:79)
2024-09-10 18:15:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:28:18 database: wrong user or password; user: "SIMONE"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:319)
	at com.mycompany.adventure.Adventure.init(Adventure.java:27)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:79)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 18 more
2024-09-10 19:28:19 database: wrong user or password; user: "SIMONE"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:29)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:79)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 17 more
2024-09-10 19:31:26 database: wrong user or password; user: "SIMONE"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:319)
	at com.mycompany.adventure.Adventure.init(Adventure.java:27)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:79)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 18 more
2024-09-10 19:31:26 database: wrong user or password; user: "SIMONE"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:29)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:79)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 17 more
2024-09-10 19:32:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:32:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO OGGETTI1 (ID, NOME, DESCRIZIONE, ALIAS) VALUES ('2', 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'ATRIO[*], 'chiave-CHIAVE');INSERT INTO OGGETTI1 (ID, NOME, DESCRIZIONE, ALIAS) VALUES ('3', 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO OGGETTI1 (ID, NOME, DESCRIZIONE, ALIAS) VALUES ('4', 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO OGGETTI1 (ID, NOME, DESCRIZIONE, ALIAS) VALUES ('5', 'Pulsante', 'Pulsante da premere per sapere se qualcuno e in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO OGGETTI1 (ID, NOME, DESCRIZIONE, ALIAS) VALUES ('6', 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO');"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO oggetti1 (id, nome, descrizione, alias) VALUES ('2', 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio, 'chiave-CHIAVE');INSERT INTO oggetti1 (id, nome, descrizione, alias) VALUES ('3', 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti1 (id, nome, descrizione, alias) VALUES ('4', 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti1 (id, nome, descrizione, alias) VALUES ('5', 'Pulsante', 'Pulsante da premere per sapere se qualcuno e in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti1 (id, nome, descrizione, alias) VALUES ('6', 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO'); [42001-200]
2024-09-10 19:32:13 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:32:13 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:33:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:33:21 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:33:21 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:33:21 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:34:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:34:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:34:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI1" not found; SQL statement:
SELECT * FROM comandi1; [42102-200]
2024-09-10 19:34:17 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:34:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:34:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE1" already exists; SQL statement:
CREATE TABLE stanze1 (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:34:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI1" not found; SQL statement:
SELECT * FROM comandi1; [42102-200]
2024-09-10 19:36:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:36:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('1', 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('2', 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('3', 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('4', 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('5', 'Pulsante', 'Pulsante da premere per sapere se qualcuno e in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('6', 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO[*]');"; SQL statement:
INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('1', 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('2', 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('3', 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('4', 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('5', 'Pulsante', 'Pulsante da premere per sapere se qualcuno e in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES ('6', 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO'); [42000-200]
2024-09-10 19:36:45 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:44:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:47:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:47:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:49:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:49:20 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:51:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:51:24 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:53:34 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:53:34 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO[*]');"; SQL statement:
INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO'); [42000-200]
2024-09-10 19:54:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:54:16 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:54:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:54:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:55:28 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:55:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" already exists; SQL statement:
CREATE TABLE stanze (id int, nome varchar(255), descrizione varchar(255), setLook varchar(500), hideDescription varchar(255)); [42101-200]
2024-09-10 19:55:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" not found; SQL statement:
SELECT * FROM comandi; [42102-200]
2024-09-10 19:55:53 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO[*]');"; SQL statement:
INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO'); [42000-200]
2024-09-10 19:59:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 19:59:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO[*]');"; SQL statement:
INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (1, 'Martello', 'Serve per attivare qualcosa', 'martello-MARTELLO');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (2, 'ChiaveBalcone', 'Serve per aprire la porta a nord dell'atrio', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (3, 'ChiaveCameraDaLetto', 'Serve per aprire la porta ad ovest della biblioteca', 'chiave-CHIAVE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (4, 'Torcia', 'Utilizzata per fare luce in una certa stanza', 'torcia-luce-TORCIA-LUCE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (5, 'Pulsante', 'Pulsante da premere per sapere se qualcuno è in bagno', 'pulsante-bottone-PULSANTE-BOTTONE');INSERT INTO oggetti (id, nome, descrizione, alias) VALUES (6, 'Letto', 'Un normale letto. Di solito lo usi per dormire o rilassarti', 'letto-LETTO'); [42000-200]
2024-09-10 19:59:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-10 20:02:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 20:02:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-10 20:03:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-10 20:03:28 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "COMANDI" already exists; SQL statement:
CREATE TABLE comandi (id int, nome varchar(255), alias varchar(500)); [42101-200]
2024-09-11 12:48:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:368)
	at com.mycompany.adventure.Adventure.init(Adventure.java:29)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 22 more
2024-09-11 12:48:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:31)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2024-09-11 12:48:26 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:368)
	at com.mycompany.adventure.Adventure.init(Adventure.java:29)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 22 more
2024-09-11 12:48:26 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:31)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2024-09-11 12:55:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-11 12:56:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-11 12:56:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-11 12:56:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-12 18:22:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-12 18:22:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-12 20:47:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-12 20:48:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-12 20:50:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-12 20:50:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-12 20:57:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-09-12 20:58:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "STANZE" not found; SQL statement:
SELECT * FROM stanze; [42102-200]
2024-09-13 21:00:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:567)
	at com.mycompany.adventure.Adventure.init(Adventure.java:31)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 22 more
2024-09-13 21:00:52 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:33)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2024-09-13 21:13:52 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.tryConnection(Adventure.java:576)
	at com.mycompany.adventure.Adventure.init(Adventure.java:31)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 22 more
2024-09-13 21:13:52 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.mycompany.adventure.Adventure.init(Adventure.java:33)
	at com.mycompany.game.Engine.<init>(Engine.java:24)
	at com.mycompany.game.Engine.main(Engine.java:80)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/simone/Documents/NetBeansProjects/Il_Nome/resources/db/IlNome.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
